modules/core/src/main/java/org/apache/synapse/core/axis2/ProxyService.java

```diff
@@ -257,6 +257,11 @@ public void setPreservePolicy(String preservePolicy) {
 
     private AxisService axisService;
 
+    /**
+     * Comment Texts List associated with the proxy service
+     */
```

Comment:
Please refer to following to learn conventions in doc comments.
https://www.oracle.com/technetwork/java/javase/tech/index-137868.html